### PR TITLE
Demo Changes Added.

### PR DESCRIPTION
**Pull request by Nikita5796**

### CODE REVIEW

**Comment by ketannaik** [comment]
Any specific reason why DLL files are being committed?

File: Services/BookService.cs [hunk 1]
```diff
@@ -52,7 +52,8 @@ public List<Book> GetBookByCategory(int categoryid)
             }
             catch (Exception ex)
             {
-                throw ex;
+                //throw ex;
+                return null;
```
Comment (bala0105) [inline Services/BookService.cs:56]: any specific reason for returning null in catch instead of throwing ex?

**Review by bala0105 — APPROVED** [review]

File: Services/BookService.cs (hunk 1 above)
Comment (Nikita5796) [reply Services/BookService.cs:56]: reason for returning null is to check in front end if result is null show error message.

**Review by ketannaik — APPROVED** [review]

File: Services/BookService.cs (hunk 1 above)
Comment (ketannaik) [reply Services/BookService.cs:56]: Send proper message instead of null

**Pull request merged**